src/promptflow/tests/executor/e2etests/test_batch_engine.py

```diff
@@ -10,12 +10,16 @@
 import pytest
 
 from promptflow._constants import OUTPUT_FILE_NAME
+from promptflow._proxy._proxy_factory import ProxyFactory
 from promptflow._sdk.entities._run import Run
 from promptflow._sdk.operations._local_storage_operations import LocalStorageOperations
 from promptflow._utils.utils import dump_list_to_jsonl
 from promptflow.batch._batch_engine import BatchEngine
 from promptflow.batch._errors import EmptyInputsData
 from promptflow.batch._result import BatchResult
+from promptflow._proxy._single_line_python_executor_proxy import SingleLinePythonExecutorProxy
+from promptflow._proxy._chat_group_orchestrator_proxy import ChatGroupOrchestratorProxy
+from promptflow._sdk.entities._chat_group._chat_role import ChatRole
```

Comment:
Please also include this in the promptflow_internal? Because runtime needs to import it.